spring-ejb/ejb-remote-for-spring/src/test/java/com/baeldung/ejb/tutorial/HelloStatefulWorldTest.java

```diff
@@ -0,0 +1,26 @@
+package com.baeldung.ejb.tutorial;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class HelloStatefulWorldTest {
+
+    private HelloStatefulWorldBean statefulBean;
+    
+    @Before
+    public void setup() {
+        statefulBean = new HelloStatefulWorldBean();
+    }
+    
+    @Test
+    public void statefulTest() {
```

Comment:
1) Test method names should follow the givenX_whenY_thenZ naming convention (givenX is optional).
2) Leave a blank line between the given part of a test (if any) and the when part, and another between the when part and the then part.
3) Use AssertJ assertions rather than the standard JUnit assertions.